staging/src/k8s.io/client-go/features/kube_features.go

```diff
@@ -41,6 +43,34 @@ const (
 	WatchList featuregate.Feature = "WatchList"
 )
 
+func DefaultFeatureGates() featuregate.FeatureGate {
```

Comment:
Do we want to implement the full `featuregate.FeatureGate` interface? 
I think it depends on how we would like to expose client-go based flags. 

Assuming these flags can be set either explicitly by calling a method or by providing an environment variable, we don't have to implement the full interface.